src/integrationTest/sam.test.ts

```diff
@@ -465,8 +439,11 @@ describe('SAM Integration Tests', async function () {
 
                     await createSamApplication(testDir)
                     appPath = path.join(testDir, samApplicationName, scenario.path)
+
                     cfnTemplatePath = path.join(testDir, samApplicationName, 'template.yaml')
+                    const readmePath = path.join(testDir, samApplicationName, 'README.md')
                     assert.ok(await fileExists(cfnTemplatePath), `Expected SAM template to exist at ${cfnTemplatePath}`)
+                    assert.ok(await fileExists(readmePath), `Expected SAM App readme to exist at ${readmePath}`)
```

Comment:
Good idea. It is often more maintainable to group related asserts in one medium-sized--or bigger--test, than to have many granular tests. And often has performance benefits too.